src/Analyzers/Microsoft.AspNetCore.Analyzer.Testing/src/DiagnosticProject.cs

```diff
@@ -25,6 +25,7 @@ public class DiagnosticProject
         /// </summary>
         public static string TestProjectName = "TestProject";
 
+        private static readonly ICompilationAssemblyResolver _assemblyResolver = new AppBaseCompilationAssemblyResolver();
```

Comment:
This uses the app base directory by default. That works locally and should work everywhere&hellip;